Makefile

```diff
@@ -1,41 +1,50 @@
-export CGO_ENABLED:=0
+# kernel-style V=1 build verbosity
+ifeq ("$(origin V)", "command line")
+       BUILD_VERBOSE = $(V)
+endif
 
-VERSION=$(shell git describe --dirty)
-LD_FLAGS="-w -X github.com/coreos/container-linux-config-transpiler/internal/version.Raw=$(VERSION)"
+ifeq ($(BUILD_VERBOSE),1)
+       Q =
+else
+       Q = @
+endif
 
+VERSION=$(shell git describe --dirty)
 REPO=github.com/coreos/container-linux-config-transpiler
+LD_FLAGS="-w -X $(REPO)/internal/version.Raw=$(VERSION)"
 
-all: build
+export GOPATH=$(shell pwd)/gopath
+export CGO_ENABLED:=0
 
-build: bin/ct
+.PHONY: all
+all: bin/ct
 
-bin/ct:
-	@go build -o $@ -v -ldflags $(LD_FLAGS) $(REPO)/internal
+gopath:
```

Comment:
As mentioned in the commit message, I'm not a fan of the global GOPATH and I don't use it on my system. There is also testing benefit to using a separate GOPATH, since you won't pull in anything from the surrounding environment.